src/joint.dia.element.js

```diff
@@ -694,8 +694,6 @@ joint.dia.ElementView = joint.dia.CellView.extend({
 
     pointerdown: function(evt, x, y) {
 
-        this.model.trigger('batch:start');
-
         // target is a valid magnet start linking
         if (evt.target.getAttribute('magnet') && this.paper.options.validateMagnet.call(this.paper, this, evt.target)) {
 
```

Comment:
I see, but I don't think it is unnecessary. The problem is that `batch:start` is triggered way to late now. Suppose you are listening to graph on `all` events and you create a link by dragging & dropping a port.

`add`, `batch:start`, `change:target`, `change:target`, `change:target`, `batch:stop` is what you see.

`batch:start`, `add`, `batch:start`, `change:target`, `change:target`, `change:target`, `batch:stop`, `batch:stop` is what you want to see (or alternatively `batch:start`, `add`, `change:target`, `change:target`, `change:target`, `batch:stop`, but an extra batch is not an issue here).

If you add some batch names it starts making more sense I think.
- cell batch:start/stop :  `{ batchName: 'pointer' }`
- element (a valid magnet branch) batch:start/stop : `{ batchName: 'add-link' }`

`batch:start(add-link)`, `add`, `batch:start(pointer)`, `change:target`, `change:target`, `change:target`, `batch:stop(pointer)`, `batch:stop(add-link)`